### DISCUSSION TITLE
How to install and configure ferretdb on Linux？

### DISCUSSION THREAD

**Discussion by gaoyan0204** [FerretDB/FerretDB#966] [General]
How to install and configure ferretdb on Linux？

**Comment by AlekSi** [comment]
Did you try our [quickstart guide](https://github.com/FerretDB/FerretDB#quickstart)?

**Reply by gaoyan0204** [reply]
I tried, but failed all the time。I need to install on Linux，How to install and configure ferretdb by ferretdb.rpm？

**Reply by gaoyan0204** [reply]
I installed it through rpm, but I don't know how to configure and start it。
